.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea/feature for this project
+---
+<!-- **Are you in the right place?**
+1. For issues or feature requests, please create an issue in this repository.
+2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://Rook-io.slack.com).
+3. Did you already search the existing open issues for anything similar? -->
+
+Is this a bug report or feature request?
+* Feature Request
+
+**Feature Request**
+
+Are there any similar features already existing:
```

Comment:
Is this really necessary? Why not just have a disclaimer at the top saying something like "Verify there are not similar features already in existence before filing the feature request."

Reply:
I'd like to leave it in because if there are similar features we can see if we can a) build upon them or b) if it is not necessary to "implement" the request because of the existing feature.

Reply:
I'm not convinced- it still seems unnecessary.

In the case of (a), that decision should be made by developers that would implement the feature. The developers should know what features exist already and shouldn't be relying on someone filing a ticket to remind them of things they could extend to get the desired effect. In the case of (b), it would be asinine to even file a feature request for something that already exists.